src/pages/StoryPage.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react'
 import Helmet from 'react-helmet-async'
 import { getBlockComponent } from '../blocks'
-import { StoryContainer } from '../storyblok/StoryContainer'
+import { BodyStory, StoryContainer } from '../storyblok/StoryContainer'
+
+const getBlocks = (story: BodyStory) =>
```

Comment:
This function name could be slightly more descriptive. How about this?

```suggestion
const getBlocksOrDefault = (story: BodyStory) =>
```